clients/venice-admin-tool/src/main/java/com/linkedin/venice/ClusterTaskRunner.java

```diff
@@ -0,0 +1,100 @@
+package com.linkedin.venice;
+
+import com.linkedin.venice.exceptions.VeniceException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.locks.ReentrantLock;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+/**
+ * This class is a simple runnable which keeps fetching task from list and execute the assigned task. The task fetching
+ * and progress tracking / checkpointing is thread-safe, so it can be run in parallel.
+ */
+public class ClusterTaskRunner implements Runnable {
```

Comment:
Can we change the name of this class to something else? ClusterTaskRunner seems misleading to me